GovUk.Frontend.Umbraco.ExampleApp/uSync/v9/ContentTypes/govukfileuploadsettings.config

```diff
@@ -1,97 +1,81 @@
-			<Type>Umbraco.TextBox</Type>
-			<Mandatory>false</Mandatory>
-			<Validation></Validation>
-			<Description><![CDATA[A comma-separated list of accepted file extensions or media types. See [the accept attribute on MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/accept).]]></Description>
-			<SortOrder>7</SortOrder>
-			<Tab Alias="settings">Settings</Tab>
-			<Variations>Nothing</Variations>
-			<MandatoryMessage></MandatoryMessage>
-			<ValidationRegExpMessage></ValidationRegExpMessage>
-			<LabelOnTop>false</LabelOnTop>
-		</GenericProperty>
-	</GenericProperties>
-	<Tabs>
-		<Tab>
-			<Key>b282eda2-24e8-4a95-8796-73238b272d53</Key>
-			<Caption>Settings</Caption>
-			<Alias>settings</Alias>
-			<Type>Group</Type>
-			<SortOrder>0</SortOrder>
-		</Tab>
-		<Tab>
-			<Key>d2bbeb2c-8bc0-4bf4-942c-88f5d54208bb</Key>
-			<Caption>Error messages</Caption>
-			<Alias>errorMessages</Alias>
-			<Type>Group</Type>
-			<SortOrder>1</SortOrder>
-		</Tab>
-	</Tabs>
+  <Info>
+    <Name>File upload settings</Name>
+    <Icon>icon-cloud-upload color-black</Icon>
+    <Thumbnail>folder.png</Thumbnail>
+    <Description>Settings for a file upload component. Ask users to upload something only if it’s critical to the delivery of your service.</Description>
+    <AllowAtRoot>False</AllowAtRoot>
+    <IsListView>False</IsListView>
+    <Variations>Nothing</Variations>
+    <IsElement>true</IsElement>
+    <HistoryCleanup>
+      <PreventCleanup>False</PreventCleanup>
+      <KeepAllVersionsNewerThanDays></KeepAllVersionsNewerThanDays>
+      <KeepLatestVersionPerDayForDays></KeepLatestVersionPerDayForDays>
+    </HistoryCleanup>
+    <Folder>GOV.UK</Folder>
+    <Compositions>
+      <Composition Key="bf69e8d6-97a6-4b50-bec7-c5993e120339">govukCssClasses</Composition>
+      <Composition Key="37cc75ea-be5c-4107-b9b8-0ef34f4b7c5e">govukErrorMessagePrefix</Composition>
+      <Composition Key="d508995b-a958-4273-8ab3-50735220a1b2">govukGrid</Composition>
+      <Composition Key="d7016563-0b6f-4c3a-b75d-b0bbe66419fd">govukGridColumnClasses</Composition>
+      <Composition Key="2bc3c434-f3ff-495d-b360-1a005245602e">govukLabelIsPageHeading</Composition>
+      <Composition Key="23e1a03d-dc42-4318-9e17-d458dde2dba6">govukModelProperty</Composition>
+      <Composition Key="f2365c7e-08c3-4ef9-abbc-e337e34de272">govukValidationRequired</Composition>
+    </Compositions>
+    <DefaultTemplate></DefaultTemplate>
+    <AllowedTemplates />
+  </Info>
+  <Structure />
+  <GenericProperties>
+    <GenericProperty>
+      <Key>02492b18-6710-4c30-872c-8019f4ba037c</Key>
+      <Name>File Type</Name>
+      <Alias>errorMessageFileType</Alias>
+      <Definition>0cc0eba1-9960-42c9-bf9b-60e150b429ae</Definition>
+      <Type>Umbraco.TextBox</Type>
+      <Mandatory>false</Mandatory>
+      <Validation></Validation>
+      <Description><![CDATA[Error message for file type validation]]></Description>
```

Comment:
I can't move Required to the top because the ordering is locked.

![image](https://github.com/user-attachments/assets/a5e0adcf-47fb-416a-a006-33502dc45b63)

Reply:
That's fine, I meant it's at the top of the error messages. Error message prefix is a little different.